internal/librarian/generate.go

```diff
@@ -51,25 +52,43 @@ func init() {
 }
 
 func runGenerate(ctx context.Context) error {
-	startTime := time.Now()
-	workRoot, err := createWorkRoot(startTime)
-	if err != nil {
+	if err := validateRequiredFlag("api-path", flagAPIPath); err != nil {
```

Comment:
The diff is being unhelpful here. Really I've just moved the checks that were at line 92 to the start of `runGenerate`.